challenge_3/python/popcorn_anachronism/src/challenge_3.py

```diff
@@ -0,0 +1,16 @@
+a = [2,2,3,7,5,7,7,7,4,7,2,7,4,5,6,7,7,8,6,7,7,8,10,12,29,30,19,10,7,7,7,7,7,7,7,7,7]
+
+def majority_element(array):
+    n = len(array)
+    array_freq = {}     #dict of frequencies of items in array
```

Comment:
I don't think my old solution was O(n^2), I ran a test case with 3570000 list elements and it gave me
time_oldversion: 0.400 seconds
time_newversion: 0.153 seconds

Regardless, Counter was more efficient and I've added that with my new commit. Thanks!